decidim-pages/app/views/decidim/pages/application/show.html.erb

```diff
@@ -9,8 +9,8 @@
   </div>
 </div>
 
-<%= content_for :expanded do %>
-  <% if @page.commentable? %>
+<% if global_configuration.comments_enabled || current_step_configuration.comments_enabled %>
```

Comment:
Could it be done some way (via delegation or a helper) so you just need to ask it once?